bin/build-windows.rb

```diff
@@ -119,14 +124,14 @@ def clone_and_rynsc_private_repo(fork, ref, hostname, ssh_key)
 puts "Build-Windows.rb... building cpp-pcp-client"
 Kernel.system("set -vx;#{scp_command} #{File.join(SCRIPT_ROOT, 'build-cpp-pcp-client.ps1')} Administrator@#{hostname}:/home/Administrator/")
 fail "Copying build-cpp-pcp-client.ps1 to #{hostname} failed" unless $?.success?
-clone_and_rynsc_private_repo(CPPPCPCLIENT['url'], CPPPCPCLIENT['ref'], hostname, ssh_key)
+clone_and_rynsc_private_repo(CPPPCPCLIENT['url'], CPPPCPCLIENT['ref'], hostname, ssh_key, 'cpp-pcp-client')
```

Comment:
Yes, this was a change that was made in the cve branch. It ultimately should be in both master and stable